source/openapi3/split.yaml

```diff
@@ -1192,7 +1192,7 @@ info:
     ```
 
     Split-Signature:
-    1514772000.93eee90206280b25e82b38001e23961cba4c007f4d925ba71ecc2d9804978635
+    1514772000.93eee90206280b25e82b38001e23961ASB4c007f4d925ba71ecc2d9804978635
```

Comment:
Bit too ambitious with the find and replace here. This signature just happens to have `cba` but doesn't have anything to do with the actual CBA

Reply:
My bad, fixed and committed